data/deprecated.json

```diff
@@ -1743,6 +1755,22 @@
     "old": {"volcano": "extinct"},
     "replace": {"volcano:status": "extinct"}
   },
+  {
+    "old": {"voltage-high": "*"},
+    "replace": {"voltage:primary":"$1"}
+  },
+  {
+    "old": {"voltage-low": "*"},
+    "replace": {"voltage:secondary":"$1"}
+  },
+  {
+    "old": {"voltage-high": "*", "transformer":"generator"},
+    "replace": {"voltage:secondary":"$1", "transformer":"generator"}
+  },
+  {
+    "old": {"voltage-low": "*", "transformer":"generator"},
+    "replace": {"voltage:primary":"$1", "transformer":"generator"}
+  },
```

Comment:
I _think_ these two should be included before line 1758, because the other deprecation rule would otherwise potentially overrule these ones.

Reply:
Good point, I've swapped them